ops/scripts/pipeline/endpoint-test.sh

```diff
@@ -44,44 +40,39 @@ while [[ $# -gt 0 ]]; do
         slot_name="${2}"
         shift 2
         ;;
-
-    --stackName)
-        stack_name="${2}"
-        shift 2
-        ;;
-
-    --priority)
-        priority="${2}"
-        shift 2
-        ;;
     *)
         exit 2 # error on unknown flag/switch
         ;;
     esac
 done
 
+webStatusCode=""
+apiStatusCode=""
+targetApiURL="https://${api_name}.azurewebsites${host_suffix}/api/healthcheck"
+targetWebAppURL="https://${webapp_name}.azurewebsites${host_suffix}"
 # shellcheck disable=SC1083 # REASON: Wants to quote http_code
-webCmd="curl -q -o -I -L -s -w "%{http_code}" --retry 5 --retry-delay 60 --retry-connrefused -f https://${webapp_name}.azurewebsites${host_suffix}"
+webCmd="curl -q -o /dev/null -I -L -s -w "%{http_code}" --retry 5 --retry-delay 60 --retry-connrefused -f ${targetWebAppURL}"
```

Comment:
We'll need to confirm if this works on the USTP side. We were notified that USTP had updated their build agent so this will most likely work.

Reply:
@magile32, okay. As it was, the `-I` flag was being ignored because the `-o` flag requires a `<file>` argument to follow it. So we were seeing odd behavior running locally and were exiting early without actually validating the healthcheck response. Similarly with the `apiCmd`, but with the additional oddity that the `-I` flag was causing a problem there.

Reply:
Original code leverage --retry-all-errors
`curl -q -o -I -L -s -w "%{http_code}" --retry 5 --retry-delay 60 --retry-all-errors -f https://${{ env.webappName }}.azurewebsites${{ vars.AZ_HOSTNAME_SUFFIX }}`

Let's modify the curl command to leverage `--retry 5 --retry-delay 60 --retry-all-errors ` instead of --retry-connfefused

Reply:
Confirmed